submissions/Neskazhuk/popup/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a class="link" href="#" tabindex="1">Hello</a></li>
+                <li class="nav-item"><a class="link" href="#" tabindex="2">World</a></li>
+                <li class="nav-item popup">
+                    <input id="popup-input-id" class="popup-input" type="checkbox" tabindex="3">
+                    <label for="popup-input-id" class="popup-btn">
+                        <img class="popup-icon" src="./icons/popup-button.png" alt="popup icon">
```

Comment:
[Read this article](https://htmlacademy.ru/blog/articles/alt-text) to find out how and when populating `alt` attribute

Reply:
The link is not available, cause it's on ru domain. 'Alt' is used for <img> tag to describe the content inside it in case of bad connection or something. So what's wrong here?

Reply:
Try [this one](http://web.archive.org/web/20220812092150/https://htmlacademy.ru/blog/articles/alt-text)

Reply:
changed popup 'alt' text to an empty string